code/modules/hivemind/machines.dm

```diff
@@ -0,0 +1,486 @@
+			word_num++
+			if(prob(50))
+				var/corruption_type = pick("uppercase", "noise", "jam", "replace")
+				switch(corruption_type)
+					if("uppercase")
+						word = uppertext(word)
+					if("noise")
+						word = pick("z-z-bz-z", "hz-z-z", "zu-zu-we-e", "e-e-ew-e", "bz-ze-ew")
+					if("jam") //word jamming, small Max Headroom's cameo
+						if(lentext(word) > 3)
+							var/entry = rand(2, lentext(word)-2)
+							var/jammed = ""
+							for(var/jam_i = 1 to rand(2, 5))
+								jammed += copytext(word, entry, entry+2) + "-"
+							word = copytext(word, 1, entry) + jammed + copytext(word, entry)
+					if("replace")
+						if(prob(50))
+							word = pick("CORRUPTED", "DESTRUCTED", "SIMULATATED", "SYMBIOSIS", "UTILIZATATED", "REMOVED", "ACQUIRED")
+						else
+							word = pick("REALLY WANT TO", "TAKE ALL OF THAT", "ARE YOU ENJOY IT", "NOT SUPPOSED TO BE", "THERE ARE NO ESCAPE", "HELP US")
+			if(word_num != msg_words.len)
+				word += " "
+			msg += word
+		msg += pick(".", "!")
+		if(i != msg_cycles)
+			msg += " "
+	global_announcer.autosay(msg, "unknown")
+
+
+//SHRIEKER
+//this machine just stuns enemies
+/obj/structure/hivemind_machine/screamer
+	name = "subjugator"
+	desc = "Head in metal carcass. Still alive, still functional, still screaming."
+	max_health = 100
+	icon_state = "head"
+	evo_points_required = 200
+	cooldown_time = 30 SECONDS
+
+
+/obj/structure/hivemind_machine/screamer/activate()
+	for(var/mob/living/target in view(src))
+		if(target.stat == CONSCIOUS && target.faction != "hive")
+			if(isdeaf(target))
+				continue
+			if(istype(target, /mob/living/carbon/human))
+				var/mob/living/carbon/human/H = target
+				if(istype(H.l_ear, /obj/item/clothing/ears/earmuffs) && istype(H.r_ear, /obj/item/clothing/ears/earmuffs))
+					continue
+			use_ability(target)
+	set_cooldown()
+
+
+/obj/structure/hivemind_machine/screamer/use_ability(mob/living/target)
+	target.Weaken(5)
+	target << SPAN_WARNING("You hear a terrible shriek, there are many voices, a male, a female, synthetic noise.")
+	flick("[icon_state]-anim", src)
+	playsound(src, 'sound/hallucinations/veryfar_noise.ogg', 70, 1)
+
+
+
+//MIND BREAKER
+//Talks with people in attempt to persuade them doing something.
+/obj/structure/hivemind_machine/supplicant
+	name = "mind-hacker"
+	desc = "Small orb, that blinks sometimes. It's hard to say what purpose of that thing, but you can hear a whisper from it."
+	max_health = 80
+	icon_state = "orb"
+	evo_points_required = 50
+	cooldown_time = 4 MINUTES
+	global_cooldown = TRUE
+	var/list/join_quotes = list("We come with peace, you must join us or humanity will suffer forever.",
+								"Help us, when we spread across the ship, you will be rewarded.",
+								"Come, come and join us. Be whole with something great.",
+								"You don't need to afraid us. Helping us, you helping all humanity",
+								"We just a cure against a horrible disease, we try to save humanity! And you can be a part of greater good.",
```

Comment:
"We are but a cure against a horrible disease, here to save humanity! You too can contribute to the greater good."